javascript/lib/api/src/options/filter.options.ts

```diff
@@ -154,7 +154,12 @@ export const Le: (property: string, value: any) => Filter =
  */
 export const In: (property: string, ...value: any[]) => Filter =
   (property: string, ...value: any[]) => {
-    return new DefaultFilter(`in(${property},${value.map(v => `"${v}"`).join()})`);
+    return new DefaultFilter(`in(${property},${value.map(v => {
+      if (typeof v === 'number' || typeof v === 'boolean') {
```

Comment:
I would suggest creating a function that maps `value:any` to the `string` representation which can be used in the filters. It can be used by `In()` and `standardFilter()`.